extern/gallery/en-US/GalleryContents.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
```

Comment:
Put this xml file and the above images in %appdata%/Dynamo/(VersionNumber)/gallery/en-US/ for trying this out